packages/popover/components/PopoverListItemIcon.tsx

```diff
@@ -1,20 +1,8 @@
 import * as React from "react";
 import Icon, { IconProps } from "../../icon/components/Icon";
 
-interface PopoverListItemIconProps extends IconProps {
-  /**
-   * Which side of the menu item the icon appears on
-   */
-  position?: "start" | "end";
```

Comment:
Same thing here with the default value for the position prop on this component as the `PopoverListItemAvatar`.